Examples/UIExplorer/MapViewExample.js

```diff
@@ -309,7 +310,7 @@ exports.examples = [
         rightCalloutView: (
           <TouchableOpacity
             onPress={() => {
-              alert('You Are Here');
+              Alert.alert('You Are Here');
```

Comment:
I'll probably revert it when I merge this. But for future reference, it's better to avoid making unrelated changes in PRs anyway, even if they are uncontroversial, as it makes it harder to review.

If we do ever want to move away from using `alert()`, we can just do it once as a codemod across the whole project.